model/id.go

```diff
@@ -24,3 +27,21 @@ func NewID() string {
 	b.Truncate(26) // removes the '==' padding
 	return b.String()
 }
+
+// ClusterNewID is a globally unique identifier for cluster ID.  It is a [A-Z0-9] string 26
+func ClusterNewID() string {
+	var b bytes.Buffer
+	encoder := base32.NewEncoder(encoding, &b)
+	encoder.Write(uuid.NewRandom())
+	encoder.Close()
+	b.Truncate(26) // removes the '==' padding
+	strID := b.String()
```

Comment:
I think we could reuse `NewID` at the beginning as this is exactly the same code:
```suggestion
	strID := NewID()
```